src/main/java/thaumicenergistics/common/parts/AEPartsEnum.java

```diff
@@ -45,11 +45,12 @@ public enum AEPartsEnum {
 
     VisInterface(ThEStrings.Part_VisRelayInterface, PartVisInterface.class),
 
-    CreativeVisInterface(ThEStrings.Part_CreativeVisRelayInterface, PartCreativeVisInterface.class),
-
     EssentiaStorageMonitor(ThEStrings.Part_EssentiaStorageMonitor, PartEssentiaStorageMonitor.class),
 
-    EssentiaConversionMonitor(ThEStrings.Part_EssentiaConversionMonitor, PartEssentiaConversionMonitor.class);
+    EssentiaConversionMonitor(ThEStrings.Part_EssentiaConversionMonitor, PartEssentiaConversionMonitor.class),
+
+    CreativeVisInterface(ThEStrings.Part_CreativeVisRelayInterface, PartCreativeVisInterface.class),
```

Comment:
Need a semi-colon here (or some line after) to fix the compilation error

Reply:
Sorry about that. Had an additional enum member on a local project and caught this mistake halfway in. Artifact of using `git add --patch`, I suppose